package/MDAnalysis/analysis/density.py

```diff
@@ -511,7 +512,13 @@ def current_coordinates():
     for ts in u.trajectory:
         print("Histograming %6d atoms in frame %5d/%d  [%5.1f%%]\r" % \
               (len(coord), ts.frame, u.trajectory.n_frames, 100.0 * ts.frame / u.trajectory.n_frames),)
-        coord = current_coordinates()
+        if sel_update=True:
```

Comment:
The syntax is not correct here. To check for equality one would use `==`. But since `sel_update` contains a boolean that is not even necessary. Just writinge `if sel_update` is enough.

```python
if sel_update:
    # do stuff
```